lesson03/assignment/index.js

```diff
@@ -0,0 +1,93 @@
+  //  The code starts from here
+  // devJSON holds four objects in an array 
+  const devJSON = [{
+    "name": "Paolo Maldini",
+    "jobTitle": "Front End Developer",
+    "company": "Web Weavers",
+    "experience": "3 years",
+    "school": "University of Washington",
+    "major": "Marketing",
+    "email": "paolo@example.com",
+    "linkedInUrl": "paolo.linkedinprofile.com",
+    "codeLanguages": [
+        "HTML", "CSS", "JavaScript", "Node", "Express" 
+    ]    
+  },
+  {
+    "name": "Barbara Bonansea",
+    "jobTitle": "Software Engineer",
+    "company": "Excellence in the Cloud",
+    "experience": "12 years",
+    "school": "University of Southern California",
+    "major": "Computer Science",
+    "email": "barbara@example.com",
+    "linkedInUrl": "barbara.linkedinprofile.com",
+    "codeLanguages": [
+        "HTML", "JavaScript", "C", "Go" 
+    ]
+  },
+  {
+    "name": "Javier Hernandez",
+    "jobTitle": "User Experience Engineer",
+    "company": "Web Sites and More",
+    "experience": "5 years",
+    "school": "Seattle University",
+    "major": "Performing Arts",
+    "email": "javier@example.com",
+    "linkedInUrl": "javier.linkedinprofile.com",
+    "codeLanguages": [
+        "HTML", "CSS" 
+    ]
+  },
+  {
+    "name": "Maribel Dominguez",
+    "jobTitle": "Front End Engineer",
+    "company": "Bits and Bytes",
+    "experience": "6 years",
+    "school": "University of Washington",
+    "major": "Mechanical Engineering",
+    "email": "maribel@example.com",
+    "linkedInUrl": "maribel.linkedinprofile.com",
+    "codeLanguages": [
+       "HTML", "CSS", "JavaScript", "React", "Vue", "Redux"
+    ]
+  }
+ ] 
+
+// console.log(devJSON);
+
+
+// devHTML calls the array of objects that applies the map method. The .map method will map 'devJSON' array. The function comes inside the parentheses with the argument 'el' (element) which represents the HTML data.
+let devHTML = devJSON.map(function(el) {
+  // In order to save line of code, I placed 'return' here instead of the bottom, otherwise I would need to create another variable 

```

Comment:
Definitely more efficient than assigning a variable and returning it. Good choice.